README.md

```diff
@@ -64,40 +66,43 @@ To get started with developing the frontend, you will need NodeJS & Python 3.
 
 Set up the Python portion of the program by running:
 
-```
+```bash
 $ virtualenv venv
 $ source venv/bin/activate
 $ pip install -e .
 ```
 
-Then to download the cached parts list and process it, run:
+Then to download the cached parts list (from the GH Pages site, generated via GH Actions) and process it, run:
 
-```
+```bash
 $ wget https://yaqwsx.github.io/jlcparts/data/cache.zip https://yaqwsx.github.io/jlcparts/data/cache.z0{1..8}
 $ 7z x cache.zip
 $ mkdir -p web/public/data/
 $ jlcparts buildtables --jobs 0 --ignoreoldstock 30 cache.sqlite3 web/public/data
 ```
 
+To understand how the GH Actions generate the cache zip file, refer to the
+[update_components.yaml workflow](.github/workflows/update_components.yaml).
+
 To launch the frontend web server, run:
 
-```
+```bash
 $ cd web
 $ npm install
 $ npm start
 ```
 
-## The Page Is Broken!
+## Reporting Issues
```

Comment:
This was intentional as BFU don't have the connection between "issue" and "not working" as programmers do. The exclamation mark and the word "Broken" were chosen on purpose to catch the users eye.

Reply:
"The Page Is Broken!" is non-standard among projects that have undergone intentional review. Consider "Requesting a Change"? Please know that your project has so many users that, if it breaks, people who know Git/GitHub will report it within 24h guaranteed.